templates/openstack/managed.html

```diff
@@ -1,725 +1,610 @@
+<section class="p-strip">
   <div class="row">
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
-      <h2>Managed private cloud, a perfect fit for multi-cloud environments</h2>
-      <p>We recommend a multi-cloud operations strategy for every company running applications at scale. The combination of public clouds and efficient, automated private infrastructure, ideally with outsourced and standardised operations under the hood, offers the best mix of latency and price across both fixed and variable workloads.
-        <a href="/engage/multi-cloud-guide">We understand why no one type of cloud architecture or infrastructure is ideal for everyone</a>
+      <p>While public clouds provide immediate access to the infrastructure-as-a-service (IaaS) offering, they are not economical in the long term and at scale. At the same time, implementing a private cloud comes with a cost.</p>
+      <p>
+        What if:
+        <ul>
+          <li>Your public cloud costs are growing with a trajectory you cannot afford?</li>
+          <li>You have no knowledge and experience to deploy and operate a cost-effective private cloud yourself?</li>
+          <li>You do not have enough resources to provide 24x7 coverage for the platform hosting your business apps?</li>
+          <li>You need a quick time-to-market and have no time to hire and train a dedicated operations team?</li>
+          <li>You have no facilities to host a private cloud?</li>
+        </ul>
       </p>
+      <p>Canonical's Managed OpenStack solves all those challenges, providing the fastest and the most cost-effective path to an OpenStack private cloud.</p>
+      <p><a href="/openstack/tco-calculator" >Get cost estimates for Managed OpenStack&nbsp;&rsaquo;</a></p>
     </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <h3>Complementary value propositions</h3>
-  </div>
-
-  <div class="row u-equal-height">
-    <div class="col-4 p-card">
-      <h4>Fully managed OpenStack</h4>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Controlled density and performance</li>
-        <li class="p-list__item is-ticked">Bare metal speed with containers</li>
-        <li class="p-list__item is-ticked">CAPEX for long term workloads</li>
-        <li class="p-list__item is-ticked">Regulatory compliance</li>
-        <li class="p-list__item is-ticked">Integrated data protection</li>
-        <li class="p-list__item is-ticked">Data sovereignty</li>
-        <li class="p-list__item is-ticked">Existing footprint</li>
-      </ul>
-    </div>
-    <div class="col-4 p-card">
-      <h4>Public cloud</h4>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Elasticity for highly variable load</li>
-        <li class="p-list__item is-ticked">Many regions for low latency</li>
-        <li class="p-list__item is-ticked">Choice of providers</li>
-        <li class="p-list__item is-ticked">OPEX as needed</li>
-      </ul>
+    <div class="col-4">
+      {{
```

Comment:
The image doesn't look clear. Could it be updated with the better resolution? Also, should it be aligned vertically centered?

Reply:
I added the new illustration